### PR TITLE
Create Team class and tests, set up test_helper, set up .gitignore

### PR DESCRIPTION
**Pull request by c-bartell**
- Created Team class and tests
- Tests passing
- Set up `test_helper.rb` with `minitest` and `simplecov`
- Set up `.gitignore` to prevent `coverage` files from commiting

### CODE REVIEW

**Review by GregJMitchell — APPROVED** [review]
Looks good. Thanks for adding simplecov.

File: test/test_helper.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'simplecov'
+SimpleCov.start
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'csv'
```
Comment (s-steel) [inline test/test_helper.rb:5]: simplecov setup to review testing

File: test/stat_tracker_test.rb [hunk 2]
```diff
@@ -1,7 +1,5 @@
-require 'minitest/autorun'
-require 'minitest/pride'
+require './test/test_helper'
 require './lib/stat_tracker'
-require "csv"
```
Comment (s-steel) [inline test/stat_tracker_test.rb:4]: This file was updated below.

**Review by s-steel — APPROVED** [review]

**Review by cpfergus1 — COMMENTED** [review]
Looks good! Did we want to use similar initializations methods like we used in game and game_teams?

**Pull request merged**